dll/win32/devmgr/lang/de-DE.rc

```diff
@@ -33,6 +33,7 @@ BEGIN
     IDS_RESOURCE_INTERRUPT "IRQ"
     IDS_RESOURCE_DMA "DMA"
     IDS_RESOURCE_PORT "E/A-Bereich"
+    IDS_COPY "Kopieren"
```

Comment:
Some menu items have mnemonic keys (&) assigned, but some don't.

Reply:
You can copy from notepad's `CMD_COPY`: https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=CMD_COPY